### ISSUE TITLE
Move Asset bulk actions bar down

### ISSUE THREAD

**Issue by newleeland** [silverstripe/silverstripe-asset-admin#735]
# Acceptance Criteria

 * Batch actions move to bottom
 * Can use the "more options" panel with right orientation (up)

# Notes

 * Already implemented in mobile
 * Should have an accessible implementation (tab order, how does a screen reader go from select to action?)

# Design Notes

At the moment there is bulk action toolbar that pops up when more than one file is selected. 
![Screen_Shot_2018-02-13_at_4_35_57_PM.png](https://images.zenhubusercontent.com/596d365561b4ca3b0a74bd86/886a3a71-7778-422a-9dd0-9ffe4566dfb9)
This is not ideal, as many highly used actions are hidden. Furthermore, much of the UI cover buttons underneath, such as Add folder and Upload buttons.

We could resolve this issues, but simply moving this bar to the bottom, such as, what is already used in the current mobile view.
![Screen Shot 2018-02-13 at 4.49.40 PM.png](https://images.zenhubusercontent.com/596d365561b4ca3b0a74bd86/9a5c080f-165c-40a4-9b98-af44ccab72c7)

**Comment by chillu** [comment]
@newleeland Please figure out if we want to show batch actions and single view actions like "save" and "publish" next to each other, or how we'd clarify that to the user

**Comment by newleeland** [comment]
had a little talk with @clarkepaul. And it should be ok just to have the action bar on the bottom, as there is enough space between them to see the separate context.
 ![image.jpg](https://images.zenhubusercontent.com/596d365561b4ca3b0a74bd86/fb3bc994-38bd-4155-a04f-964f2aff1d81)

**Comment by flamerohr** [comment]
Done as part of https://github.com/silverstripe/silverstripe-asset-admin/issues/693
